specifyweb/frontend/js_src/lib/components/PickLists/TreeLevelPickList.tsx

```diff
@@ -96,7 +96,11 @@ export function TreeLevelComboBox(props: DefaultComboBoxProps): JSX.Element {
                 )
               : undefined
           )
-          .then((items) => (destructorCalled ? undefined : setItems(items))),
+          .then((items) => {
+            if (destructorCalled) return undefined;
+            resource.set('definitionItem', items?.slice(-1)[0]?.value || '');
```

Comment:
is this essentially setting the default value?
if so take a look [at line 116 in this file](https://github.com/specify/specify7/pull/4462/files#diff-f4cb9250846063d72449e6534d14cf141a4a4ded271dfb63c02a9ffd665ef2c7R116)

```
resource.set('definitionItem', props.defaultValue ?? items?.slice(-1)[0]?.value || '');
```

(and add props.defaultValue to dependency array)
(and remove the line 116 because then it would be needless. or no? can you check?)

Reply:
yep seems to work